config/application.yml.example

```diff
@@ -20,8 +20,9 @@ development:
   newrelic_license_key: 'xxx'
   requests_per_ip_limit: '300'
   requests_per_ip_period: '300'
-  sms_disabled: 'true'
+  sms_disabled: true
   saml_passphrase: 'trust-but-verify'
+  saml_encryption_disabled: true
```

Comment:
why not `saml_encryption_enabled: false` ?

I.e. if the default behavior is "enabled == true" then this reads like a double negative to me (harder to grok).